doc/collections/_adrs/0004--decision_for_tools_dir.jekyll

```diff
@@ -0,0 +1,81 @@
+
```

Comment:
```suggestion
---
date: 2024-01-25
---
```

Reply:
lots deliberation went into using `.jekyll` extension for these (not my first choice), but Jekyll Markdown is actually some specific amalgamate that includes:

- Standard Markdown
- Liquid Templating
- Front-matter (as seen above)
- Perhaps other things i've likely forgotten

Reply:
regarding this change of file extension: the validation tooling is still currently oblivious about these files, and they're not yet getting checked by our validation tooling (!)